nix/release-files-ubuntu-latest.nix

```diff
@@ -9,4 +9,5 @@ pkgs.runCommandNoCC "motoko-release-${common.releaseVersion}" { } ''
   cp ${common.as_js "moc" packages.js.moc} $out/moc-${common.releaseVersion}.js
   cp ${common.as_js "moc-interpreter" packages.js.moc_interpreter} $out/moc-interpreter-${common.releaseVersion}.js
   tar --exclude=.github -C ${pkgs.sources.motoko-base-src} -czvf $out/motoko-base-library.tar.gz .
+  tar --exclude=.github -C ${pkgs.sources.motoko-core-src} -czvf $out/motoko-core.tar.gz .
```

Comment:
@rvanasa I think we just need it here.
The other os targets are check that moc compiles on these systems.
But the final release step just works on ubuntu latest and there we need to have base and core